test/weex/cases/recycle-list/components/editor.vue

```diff
@@ -0,0 +1,31 @@
+<template recyclable="true">
+  <div>
+    <text class="output">{{output}}</text>
+    <input class="input" type="text" v-model="output" />
+  </div>
+</template>
+
+<script>
+  module.exports = {
+    props: ['message'],
+    data () {
+      return {
+        output: this.message | ''
```

Comment:
`this.message || ''`?

Reply:
It's my fault! I will fix it later. Maybe you can also send a PR to fix it. 😄